src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,23 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
```

Comment:
> that was categorized as a bug.

but it's something we can't fix without changing the behavior of NServiceBus regarding component registration. That's why I think @andreasohlund has a good point and why I'd suggest to fix it without being backward compatible in 5.3. Honestly I don't see enough value fixing it in backward compatible way.

@Particular/nservicebus-maintainers please chime in